### ISSUE TITLE
 Uses Passive Event Listeners to Improve Scrolling Performance

### ISSUE THREAD

**Issue by zwantE** [mattbryson/TouchSwipe-Jquery-Plugin#350]
Google Chrome Audits returns an Error:
"Uses Passive Event Listeners to Improve Scrolling Performance"

More Information -> https://developers.google.com/web/tools/lighthouse/audits/passive-event-listeners

**Comment by YenHub** [comment]
I have found also that this leads to console errors as such:

> [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/features/6662647093133312

Scrolling performance, using this plugin on desktop devices, is laggy and leaves the UI feeling at times unresponsive and glitchy as the scroll fights the wheel steps down the page.

Commenting out the following function improves scrolling performance and dramatically smooths out the scrolling:- 

```javascript
function wheel(event) {
    if (!initDone) {
        init();
    }
	// This is the problematic section of code
    // var target = event.target;
    // var overflowing = overflowingAncestor(target);
    // if (!overflowing || event.defaultPrevented || isNodeName(activeElement, "embed") || (isNodeName(target, "embed") && /\.pdf/i.test(target.src))) {
    //     return true;
    // }
    // var deltaX = event.wheelDeltaX || 0;
    // var deltaY = event.wheelDeltaY || 0;
    // if (!deltaX && !deltaY) {
    //     deltaY = event.wheelDelta || 0;
    // }
    // if (!options.touchpadSupport && isTouchpad(deltaY)) {
    //     return true;
    // }
    // if (Math.abs(deltaX) > 1.2) {
    //     deltaX *= options.stepSize / 120;
    // }
    // if (Math.abs(deltaY) > 1.2) {
    //     deltaY *= options.stepSize / 120;
    // }
    // scrollArray(overflowing, -deltaX, -deltaY);
    //event.preventDefault();
    }
```

I am unsure at this stage what effect this will have on mobile devices scrolling abilities. 

I may opt to implement a device type test before allowing this particular section of code to run, therefore reclaiming scroll performance on Desktop whilst preserving functionality on touch devices.

**Comment by rsmith4321** [comment]
I've noticed this is still an issue. Will this be fixed in a future version? Thanks!

**Comment by mattbryson** [comment]
Yeah, I'm going to carve out some time next week to do an overhaul and
clean up alot of issues like this.

**Comment by holoduke** [comment]
Is this issue fixed?
